### DISCUSSION TITLE
Network v14 `Chocolate` Upgrade and Lotus Releases Timeline

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/lotus#7429] [General Q&A]
If you have any questions wrt the [Network v14 `Chocolate` Upgrade and Lotus Releases Timeline](https://github.com/filecoin-project/lotus/discussions/7431#discussion-3604162), leave a comment here!

**Comment by coder-chenda** [comment]
Is it OK after the lotus upgrade is successful? I have successfully upgraded to v1.13.0. If I wait for network V14 'chocolate' to upgrade automatically on October 26, what else need to be done
